plugin/cassandra/plugin.go

```diff
@@ -381,58 +480,63 @@ func (p CassandraPlugin) Backup(endpoint plugin.ShieldEndpoint) error {
 
 	info, err := os.Lstat(cassandra.DataDir)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "@R{\u2717 Recursive hard-link snapshot files}\n")
+		fmt.Fprintf(os.Stderr, "@R{\u2717 Recursive hard-link snapshot files in temp dir}\n")
```

Comment:
Can we spell out "temporary directory", everywhere we message it to the operator?